app-office/moneydance/files/moneydance-bin

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Options
+JAVA_OPTS="-client -Dawt.useSystemAAFontSettings=gasp -Dawt.useSystemAAFontSettings=on -Xmx2048m"
+MONEYDANCE_LIBS="/usr/share/moneydance/lib"
+OPENJDK=( "/opt/openjdk-bin-11.*" "/usr/lib*/openjdk-11" )
```

Comment:
I've suggested `set --` precisely because POSIX shell does not support arrays.

Reply:
Ok.